lib/turn.rb

```diff
@@ -0,0 +1,26 @@
+class Turn
+
+  attr_reader :guess,
+              :card
+
+  def initialize(guess, card)
+    @guess = guess
+    @card = card
+  end
+
+  def correct?
+    if @guess == @card.answer
+      true
+    else
+      false
+    end
+  end
+
+  def feedback
+    if correct? == true
```

Comment:
similarly here, since `correct?` returns true or false automatically, we don't need to have this boolean operator. So, this line could just read `if correct?`